src/components/themed/FilledTextInput.tsx

```diff
@@ -196,9 +196,9 @@ export const FilledTextInput = React.forwardRef<FilledTextInputRef, FilledTextIn
     if (inputRef.current != null) inputRef.current.blur()
   }
   function clear(): void {
+    if (blurOnClear || valueRef.current !== '') blur()
```

Comment:
I think this should be `valueRef.current === ''`, to match how it worked before.